packages/eas-cli/src/build/utils/url.ts

```diff
@@ -13,3 +14,13 @@ export function getBuildLogsUrl({
 export function getArtifactUrl(artifactId: string): string {
   return `${getExpoWebsiteBaseUrl()}/artifacts/${artifactId}`;
 }
+
+export function getInstallUrl(build: BuildFragment): string | null | undefined {
```

Comment:
That function might be confusing here because both types and name suggest that it will work with any builds, but in only make sense for internal ones. Maybe it would be cleaner to keep the function that constructs ios URL here and moving that  `if` statement to the `printBuildResult` function

Reply:
Yeah I agree -- I improved the naming of this function. Thank you for the review!